### PR TITLE
[plugin.video.tv3.cat] 1.3.8

### PR DESCRIPTION
**Pull request by jqandreu**
### Description
Fixed web server changes and updated Kodi version

### Checklist:

- [x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.tv3.cat/default.py [hunk 1]
```diff
@@ -0,0 +1,1109 @@
...
+                except AttributeError as e:
+                    xbmc.log("listVideos--bucle addVideo--Exception AtributeError: " + str(e))
+                   
+                except KeyError as e:
+                    xbmc.log("listVideos--bucle addVideo--Exception KeyError: " + str(e))
+                    
+                except Exception as e:
+                    xbmc.log("listVideos--bucle addVideo--Exception: " + str(e))
+                    
+            else:
```
[the remark this answers is not in the document]
Comment (jqandreu) [reply plugin.video.tv3.cat/default.py]: Fixed error in clause else. Removed one tab. This clause belongs to if statetment, not for loop

[the remark this answers is not in the document]
Comment (jqandreu) [reply plugin.video.tv3.cat/default.py]: Fixed error with else clause. This clause belongs to if statement, not for loop

**Comment by enen92** [comment]
@jqandreu pull request is incorrect.

**Comment by jqandreu** [comment]
What is incorrect?

**Comment by Rechi** [comment]
Your PR branch (plugin.video.tv3.cat) has to be based on isengard branch of this repo, as you want to submit a change to that branch.

**Comment by jqandreu** [comment]
I'm new to git. How can I fix it? Do I have to create a new branch based on the remote branch

**Comment by Rechi** [comment]
I think it is easier to create a new branch based on the remote branch.
Actually your addon is in gotham not in helix like I initially thought.

**Pull request closed without merging**